scenarios/pserranoa/openvpn.yml

```diff
@@ -0,0 +1,14 @@
+# openvpn bruteforce detection / auth_failed
+type: leaky
+name: pserranoa/openvpn
+description: "Detect openvpn bruteforce"
+filter: "evt.Meta.log_type == 'auth_failed'"
```

Comment:
Could we match `evt.Meta.service == 'openvpn' && evt.Meta.log_type == 'auth_failed'` since there may be the case that a `log_type` may be set to `auth_failed` for other services?